understanding/21/reflow.html

```diff
@@ -12,68 +12,245 @@ <h1>Understanding Reflow</h1>
+              <p>While an alternate view can also exist to show each changed line stacked on top of each other, the two-column comparison view is maintained to allow for a more direct comparison between the original and modified lines of code, which some users may find easier to understand - particularly for instances where code diffs can take up multiple lines and thus could require much more scrolling to compare individual lines to each other.</p>
+          </figcaption>
+        </figure>
+
+        <p>Finally, interfaces which represent the creation of or rendering of presentations created by users (for instance, creating or presenting slide decks) often cannot simply reflow content, as doing so could easily result in breakages to the presentation of the author's content. Instead, it would be expected that such interfaces provided content authors with a means to create an alternative presentation that does meet the requirements of this success criterion.</p>
       </section>
+
       <section>
-        <h3>Avoiding scrolling in horizontally and vertically written languages</h3>
-        <p>The success Criterion applies to both horizontally and vertically written languages. Zooming the page for horizontally written languages where pages scroll vertically by default (e.g. English) should not require horizontal scrolling. Zooming the page for vertically written languages which scroll horizontally by default should not require vertical scrolling.</p>
+        <h3>Why specifically 320px and 256px?</h3>
+        <p>The value of 320 CSS pixels for vertically scrolling content was chosen as a reasonable minimum size that authors can achieve. This value aligned with reported viewport width of small displays of common mobile devices that were available when this criterion was originally drafted. The 320 CSS pixels generally corresponds to a desktop browser window set to a width of 1280px and the browser viewport then zoomed to 400%. It should be noted that 400% applies to the dimension, not the area. It means four times the default zoom level viewport width and four times the default zoom level height.</p>
+        
+        <figure id="figure-CSS-pixel-letter-4x">
+          <img src="img/css-pixel-by-device.png" width="462" height="198" alt="Diagram showing the size of character needed by viewing distance to make the same image on the retina with small screen devices close, large screen devices further away.">
+          <figcaption>A letter of the same CSS pixel size on different displays with different resolutions</figcaption>
+        </figure>
+
+        <div class="note">
+          <p>While a 1280 by 1024 CSS pixel viewport size, zoomed by 400%, neatly aligns to the 320 and 256 CSS pixels sizing requirements of this success criterion, this success criterion is not mandating that all content fit within this exact viewport size, nor that all users who would benefit from Reflow would all be viewing web pages at this exact screen resolution.</p>
```

Comment:
It's not a major issue, but the phrase "this success criterion is not mandating that all content fit within this exact viewport size" seems out of context here, since we've already addressed the exceptions earlier. This might lead someone to wonder, "Is there something else, beyond the exceptions mentioned above?" We could either remove it or, as in previous sections, provide more context (e.g., by adding an anchor to the exceptions, as shown in the proposal below).

```suggestion
          <p>While a 1280 by 1024 CSS pixel viewport size, zoomed by 400%, neatly aligns to the 320 and 256 CSS pixels sizing requirements of this success criterion, this success criterion is not mandating that all content fit within this exact viewport size as explained in the <a href="#exceptions-and-content-considerations-for-reflow">Exceptions and Content Considerations for Reflow</a>. Additionally, not all users who could benefit from Reflow will be viewing web pages at this exact screen resolution.</p>
```